src/app/Controllers/ParameterValidator.cs

```diff
@@ -174,11 +177,67 @@ public static bool MovieId(string movieId, out string message)
                 !int.TryParse(movieId.Substring(2), out int val) ||
                 val <= 0)
             {
-                message = "Invalid Movie ID parameter";
-                return false;
+                return GetAndLogBadParam("Invalid Movie ID parameter", method, logger);
+            }
+
+            return null;
+        }
+    }
+
+    public static class ResultHandler
+    {
```

Comment:
Should this class be in it's own file? I dont have a strong opinion, but would be easier to find.

Reply:
fixed